crates/ursa-application/Cargo.toml

```diff
@@ -22,13 +22,18 @@ tracing-subscriber.workspace = true
 tracing-error.workspace = true
 ethers.workspace = true
 
-abci-rs = { version = "0.11.3", features = ["async-api"] }
-revm = { version = "3.1.0", features = [
-    "serde",
+async-abci = { version = "0.1.8", default-features = false, features = [
+    "tokio-backend",
 ] }
+abci-rs = { version = "0.11.3", features = ["async-api"] }
+
+revm = { version = "3.1.0", features = ["serde"] }
 hex = "0.4.3"
 tendermint-abci = { version = "0.23.7", features = ["client"] }
 tendermint-proto = "0.23.7"
 reqwest = "0.11.11"
+parking_lot = "0.12.1"
 ursa-utils = { path = "../ursa-utils" }
-
+prost = { version = "0.10.4", default-features = false, features = [
```

Comment:
`prost` requires us to install protoc (C++) and it would make our software lighter if we could remove it. Could we please consider using a pure rust alternative like [`quick-protobuf`](https://docs.rs/quick-protobuf/latest/quick_protobuf/)? For the most part it performs better than `prost` https://github.com/tafia/quick-protobuf/tree/master/perftest#results.

Reply:
Im going to look into this more. Im actually only bringing in prost for its message trait here, since the ABCI server crate uses prost. We are already depend on Prost in a lot of our our crates(like Narwhal, Libp2p) so im not  sure it would help with being lighter. But I could be down for making our own ABCI server implementation if it is significant performance increase